src/index.js

```diff
@@ -138,6 +138,7 @@ function configureToMatchImageSnapshot({
   storeReceivedOnFailure: commonStoreReceivedOnFailure = false,
   customReceivedDir: commonCustomReceivedDir,
   customDiffDir: commonCustomDiffDir,
+  onlyDiff: commonOnlyDiff,
```

Comment:
I think you need your default value here, else it will be set to undefined if not specified